### ISSUE TITLE
[Question] after updatting to 0.12.0 package orders, order_lines and transactions failed 

### ISSUE THREAD

**Issue by TomaszE** [fivetran/dbt_shopify#80]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

After we updated dbt package from 0.11.0 to 0.12.0 3 tables started to failing ( an hour earlier, on older version dbt run completed successfully )

### Relevant error log or model output

```shell
10:57:30
10:57:30    Database Error in model shopify__transactions (models/shopify__transactions.sql)
  000904 (42000): SQL compilation error: error line 3 at position 16
  invalid identifier 'TRANSACTIONS_UNIQUE_ID'
  compiled Code at target/run/shopify/models/shopify__transactions.sql
10:57:30  
10:57:30
10:57:30    Database Error in model shopify__order_lines (models/shopify__order_lines.sql)
  000904 (42000): SQL compilation error: error line 3 at position 16
  invalid identifier 'ORDER_LINES_UNIQUE_KEY'
  compiled Code at target/run/shopify/models/shopify__order_lines.sql
10:57:30  
10:57:30
10:57:30    Database Error in model shopify__orders (models/shopify__orders.sql)
  000904 (42000): SQL compilation error: error line 3 at position 16
  invalid identifier 'ORDERS_UNIQUE_KEY'
  compiled Code at target/run/shopify/models/shopify__orders.sql
10:57:30  
10:57:30
10:57:30  Done. PASS=352 WARN=0 ERROR=3 SKIP=7 TOTAL=362
```

### Expected behavior

to run without issues 

### dbt Project configurations

```
vars:
 shopify_database: raw
  shopify_union_schemas: ['shopify_uk','shopify_ca','shopify_fr']
  shopify_using_fulfillment_event: true # false by default
  shopify_source:
    order_pass_through_columns:
      - name: "region"
        alias: "region"
        transform_sql: "case when source_relation = 'shopify_uk' then 'UK' when source_relation = 'shopify_fr' then 'FR' else 'CA' end"
    order_line_pass_through_columns:
      - name: "region"
        alias: "region"
        transform_sql: "case when source_relation = 'shopify_uk' then 'UK' when source_relation = 'shopify_fr' then 'FR' else 'CA' end"
    customer_pass_through_columns:
      - name: "region"
        alias: "region"
        transform_sql: "case when source_relation = 'shopify_uk' then 'UK' when source_relation = 'shopify_fr' then 'FR' else 'CA' end"
    order_line_refund_pass_through_columns:
      - name: "region"
        alias: "region"
        transform_sql: "case when source_relation = 'shopify_uk' then 'UK' when source_relation = 'shopify_fr' then 'FR' else 'CA' end"
    daily_shop_pass_through_columns:
      - name: "region"
        alias: "region"
        transform_sql: "case when source_relation = 'shopify_uk' then 'UK' when source_relation = 'shopify_fr' then 'FR' else 'CA' end"
    customer_cohorts_pass_through_columns:
      - name: "region"
        alias: "region"
        transform_sql: "case when source_relation = 'shopify_uk' then 'UK' when source_relation = 'shopify_fr' then 'FR' else 'CA' end"
#    shopify_timezone: "case when source_relation = 'shopify_uk' then 'Europe/London' when source_relation = 'shopify_fr' then 'Europe/Paris' else 'America/Toronto' end"


models: 
 shopify:
    +schema: shopify # leave blank for just the target_schema
    intermediate:
      +schema: staging # leave blank for just the target_schema
  shopify_source:
    +schema: staging # leave blank for just the target_schema


```

### Package versions

  - package: fivetran/shopify
    version: [">=0.12.0", "<0.13.0"]
  

### What database are you using dbt with?

snowflake

### dbt Version

latest

### Are you willing to open a PR to help address this issue?

- [ ] Yes.
- [ ] Yes, but I will need assistance and will schedule time during our [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance
- [X] No.

**Comment by fivetran-catfritz** [comment]
Hi @TomaszE you'll need to run a `dbt run --full-refresh` after upgrading since those columns were added as part of the update and dbt most likely thinks you're on an incremental run if you didn't do the full refresh. I included more details in the Breaking Changes section of the CHANGELOG [here](https://github.com/fivetran/dbt_shopify/releases/tag/v0.12.0). 

Let me know if that doesn't resolve your issue, however!
